src/props.shadows.js

```diff
@@ -38,23 +39,32 @@ const Shadows = {
     0 41px 33px -2px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 6%)),
     0 100px 80px -2px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 7%))`,
 
-  '--inner-shadow-0': 'inset 0 0 0 1px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-  '--inner-shadow-1': 'inset 0 1px 2px 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-  '--inner-shadow-2': 'inset 0 1px 4px 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-  '--inner-shadow-3': 'inset 0 2px 8px 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-  '--inner-shadow-4': 'inset 0 2px 14px 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-}
+  "--inner-shadow-0":
+    "inset 0 0 0 1px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))",
+  "--inner-shadow-1": `
+    inset 0 -.5px 0 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))`,
+  "--inner-shadow-2": `
+    inset 0 -.5px 0 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))`,
+  "--inner-shadow-3": `
+    inset 0 -.5px 0 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))`,
+  "--inner-shadow-4": `
+    inset 0 -.5px 0 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))`,
+};
```

Comment:
looks like another typo here because the inner shadows are now *only* the highlight and don't have the inner shadow part

<img width="973" alt="Screenshot 2023-08-29 at 3 41 37 PM" src="https://github.com/argyleink/open-props/assets/1134620/847e0d2d-53f3-4559-a048-a4f0af5e9cd1">

Reply:
just to be clear too, this PR should really only be changing these lines. need to combine the highlights like you added above, with the previous shadows that were there. thinking about it too, the highlight could be another custom property, which would help us not repeat the same highlight shadow over and over.